queries/vue/injections.scm

```diff
@@ -49,6 +49,30 @@
   (#eq? @_ts "ts")
```

Comment:
If you are only meaning to changing to add `lang="tsx"` to inject typescript, then this should suffice
```suggestion
  (#eq? @_ts "ts")
  (#any-of? @_ts "ts" "tsx")
```

Reply:
👌